<p>There is no such special method <code>__raise__</code> (at least none that I have ever heard of or that I can find in <a href="http://docs.python.org/reference/datamodel.html" rel="nofollow">the Python documentation</a>).</p>

<p>Why do you want to do this?  I can't think of any reason why you want custom code be be executed when the exception is raised (as opposed to either when the exception is constructed, which you can do with the <code>__init__</code> method, or when the exception is caught, which you can do with an <code>except</code> block).  What is your use case for this behavior, and why do you expect that Python supports it?</p>
